src/style.css

```diff
@@ -1 +1,45 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars__container {
+  width: 96px;
+  height: 116px;
+  border-radius: 0.5px;
```

Comment:
The `border-radius` value of `0.5px` is very small and might not be noticeable. If you intended to create a rounded effect, consider using a larger value.